### PR TITLE
Added error code info

### PR DESCRIPTION
**Pull request by joinimran**
If users doesn't have permission to join AAD, they will also get an error 0x801C03ED. Added instructions for the same. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8971

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ If the error occurs again, check the error code against the following table to s
...
-| 0x801C03ED | Multi-factor authentication is required for a 'ProvisionKey' operation, but was not performed. <br><br> -or- <br><br> Token was not found in the Authorization header. <br><br> -or- <br><br> Failed to read one or more objects. <br><br> -or- <br><br> The request sent to the server was invalid. | Sign out and then sign in again. If that doesn't resolve the issue, unjoin the device from Azure Active Directory (Azure AD) and rejoin.
+| 0x801C03ED | Multi-factor authentication is required for a 'ProvisionKey' operation, but was not performed. <br><br> -or- <br><br> Token was not found in the Authorization header. <br><br> -or- <br><br> Failed to read one or more objects. <br><br> -or- <br><br> The request sent to the server was invalid. <br><br> -or- <br><br> User does not have permissions to join the Azure Active Directory (Azure AD) | Sign out and then sign in again. If that doesn't resolve the issue, unjoin the device from Azure Active Directory (Azure AD) and rejoin. <br> Allow user(s) to join to Azure Active Directory (Azure AD) under Azure Active Directory (Azure AD) Device settings.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:68]:
```suggestion
| 0x801C03ED | Multi-factor authentication is required for a 'ProvisionKey' operation, but was not performed. <br><br> -or- <br><br> Token was not found in the Authorization header. <br><br> -or- <br><br> Failed to read one or more objects. <br><br> -or- <br><br> The request sent to the server was invalid. <br><br> -or- <br><br> User does not have permissions to join to Azure AD. | Sign out and then sign in again. If that doesn't resolve the issue, unjoin the device from Azure  AD and rejoin. <br> Allow user(s) to join to Azure AD under Azure AD Device settings.
```

**Comment by joinimran** [comment]
Thanks, @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by mapalko — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Rich Diff view** for easier review: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9146/files?short_path=5abe610#diff-5abe610164e06ff62dfdab188f230aa4eead645c3c772f5a7ae397422f6bb16a

**Comment by illfated** [comment]
#sign-off

**Pull request merged**